src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingPreview.jsx

```diff
@@ -31,67 +30,57 @@ import TitleDesign from '../../../plots-and-tables/components/TitleDesign';
 import FontDesign from '../../../plots-and-tables/components/FontDesign';
 import LegendEditor from '../../../plots-and-tables/components/LegendEditor';
 import LabelsDesign from '../../../plots-and-tables/components/LabelsDesign';
+import { filterCells } from '../../../../../../utils/plotSpecs/generateEmbeddingCategoricalSpec';
+import { loadCellSets } from '../../../../../../redux/actions/cellSets';
 
 const { Panel } = Collapse;
 
-const EmbeddingPreview = (props) => {
-  const { experimentId } = props;
-
-  const dispatch = useDispatch();
-
-  const [selectedSpec, setSelectedSpec] = useState('cellCluster');
-  const [plotSpec, setPlotSpec] = useState({});
-  const [config, setConfig] = useState(null);
-
-  const embeddingSettings = useSelector((state) => state.experimentSettings.processing.configureEmbedding);
-  const embeddingMethod = embeddingSettings?.embeddingSettings?.method;
-  const clusteringMethod = embeddingSettings?.clusteringSettings?.method;
-  const embedding = useSelector((state) => state.embeddings);
+const EmbeddingPreview = () => {
+  const router = useRouter();
+  const { experimentId } = router.query;
+  const [selectedPlot, setSelectedPlot] = useState('sample');
+  const [plot, setPlot] = useState(false);
```

Comment:
This change is causing staging to fail, because `experimentId` is initially undefined. The `master` branch specifically has a special way of rendering this component to ensure it has the proper `experimentId` passed in once that is available to the component.

Essentially, you should replace

```
const EmbeddingPreview = () => {
  const router = useRouter();
  const { experimentId } = router.query;
...
```

with:

```
const EmbeddingPreview = (props) => {
  const { experimentId } = props;
...
```

and things should go back to normal.

Reply:
Changed this in the new commit